packages/kbn-rule-data-utils/src/alerts_as_data_rbac.ts

```diff
@@ -31,7 +31,7 @@ export const mapConsumerToIndexName: Record<AlertConsumers, string | string[]> =
   logs: '.alerts-observability.logs',
   infrastructure: '.alerts-observability.metrics',
   observability: '.alerts-observability',
-  siem: ['.alerts-security.alerts', '.siem-signals'],
+  siem: '.alerts-security.alerts',
```

Comment:
`.siem-signals` is dynamically named and also aliased to `.alerts-security.alerts-space` so no need to query both.